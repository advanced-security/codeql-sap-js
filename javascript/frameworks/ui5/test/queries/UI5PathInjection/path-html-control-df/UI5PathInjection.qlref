UI5PathInjection/UI5PathInjection.ql