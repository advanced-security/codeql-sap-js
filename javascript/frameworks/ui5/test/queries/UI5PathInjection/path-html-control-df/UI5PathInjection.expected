nodes
| webapp/controller/app.controller.js:10:17:10:27 | input: null |
| webapp/controller/app.controller.js:16:39:16:66 | oModel. ... input') |
| webapp/view/app.view.xml:5:5:7:28 | value={/input} |
| webapp/view/app.view.xml:8:5:8:37 | content={/output} |
edges
| webapp/controller/app.controller.js:10:17:10:27 | input: null | webapp/controller/app.controller.js:16:39:16:66 | oModel. ... input') |
| webapp/controller/app.controller.js:10:17:10:27 | input: null | webapp/view/app.view.xml:5:5:7:28 | value={/input} |
| webapp/controller/app.controller.js:12:26:12:45 | new JSONModel(oData) | webapp/view/app.view.xml:8:5:8:37 | content={/output} |
| webapp/view/app.view.xml:5:5:7:28 | value={/input} | webapp/controller/app.controller.js:10:17:10:27 | input: null |
| webapp/view/app.view.xml:5:5:7:28 | value={/input} | webapp/controller/app.controller.js:12:26:12:45 | new JSONModel(oData) |
#select
| webapp/controller/app.controller.js:16:39:16:66 | oModel. ... input') | webapp/view/app.view.xml:5:5:7:28 | value={/input} | webapp/controller/app.controller.js:16:39:16:66 | oModel. ... input') | The path of a saved file depends on a $@. | webapp/view/app.view.xml:5:5:7:28 | value={/input} | user-provided value |
